config/allowed_licenses.json

```diff
@@ -242,10 +234,10 @@
     },
     {
       "moduleName": "commons-codec:commons-codec",
-      "moduleVersion": "1.15",
+      "moduleVersion": "1.16.0",
       "moduleLicense": "Apache-2.0",
-      "moduleCopyright": "2002-2020, The Apache Software Foundation",
-      "licenseUrl": "https://github.com/apache/commons-codec/blob/rel/commons-codec-1.15/NOTICE.txt",
+      "moduleCopyright": "2002-2023 The Apache Software Foundation",
+      "licenseUrl": "https://github.com/apache/commons-codec/blob/rel/commons-codec-1.16.0/NOTICE.txt",
```

Comment:
https://github.com/apache/commons-codec/blob/rel/commons-codec-1.16.0/LICENSE.txt

Reply:
resolved in [9aeb2c1](https://github.com/jenkinsci/ecutest-plugin/pull/690/commits/9aeb2c12c224ff250cc72a24b8015930fb3ba03c)